help-docs/marketplace/apps/proof-of-humanity.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 11
+title: Proof of Humanity
+---
+
+### About this app
+
+Rarimo’s Proof-of-Humanity plug-in ensures that only verified humans are present either within a server or channel. It keeps the community server spam and scam-free, which saves mods’ time, improves the quality of conversation, and helps protect airdrops.
+
+The solution is uniquely robust yet does not require users to reveal any personal details. Users can select between five identity providers to verify their humanity and will receive a verified human credential that they can use across Web3 for perks and exclusive access.
+
+### Getting Started
+
+To get started:
+
+1. Install the Proof of Humanity Miniapp from the [Collab.Land Marketplace](https://cc.collab.land/).
+1. Create a discord role for verified humans.
+1. Create the following token-gating rule in the Command Center:
+
+    | Field  | Value |
+    |---|---|
+    |Chain|`Polygon`|
+    |Token Type|`ERC721`|
+    |Addresses|`0xaD7De01cb7eaAFf3a419A0a0a3133a964cD90373`|
+
+1. Create a role-exclusive channel for verified humans.
+1. Ask users to verify their humanity using the `/verify` command, ideally in the `#collabland-join` channel.
+
+<!-- TODO: replace with the prod link once the miniapp is live -->
+For detailed instructions, see [Protecting your Discord server from bots with Proof of Humanity and Collab.Land
+](https://staging.docs.rarimo.com/how-to-guides/proof-of-humanity-collabland-discord) guide.
+
+### Verifying Your Humanity
+
+1. Call the `/verify` command and follow the instructions.
+1. Complete the verification process at [Verify Your Humanity](https://robotornot.rarimo.com).
+1. Claim the role by verifying your assets in the `#collabland-join` channel.

```

Comment:
same process as usual {link}

Reply:
Could you elaborate?

Reply:
that was a note for me to add something then i forgot to add it 🤣 my b